### PR TITLE
Add --no-onchain-storage-access flag

### LINKED ISSUE #586: Remove the -l flag dependency for the onchain contract lookup
## Description
currently -l flag is being used for getting onchain data of mythril, which might lead to confusion.so remove that dependency.

### PR DESCRIPTION
**Pull request by norhh**
added a --no-onchain-storage-access 
when turned on: onchain storage lookup is turned off
else: onchain storage lookup happens

also the -l flag is now used for contract loading only.
fixes #586

### CODE REVIEW

File: mythril/laser/ethereum/call.py [hunk 1]
```diff
@@ -122,9 +122,9 @@ def get_callee_account(global_state, callee_address, dynamic_loader):
 
     try:
         code = dynamic_loader.dynld(environment.active_account.address, callee_address)
-    except Exception:
-        logging.debug("Unable to execute dynamic loader.")
-        raise ValueError()
+    except ValueError as error:
+        logging.debug("Unable to execute dynamic loader because: {}".format(str(error)))
+        raise ValueError(str(error))
```
Comment (JoranHonig) [inline mythril/laser/ethereum/call.py]: Can you do `raise` here, that should allow us to keep the stack trace

File: mythril/support/loader.py [hunk 2]
```diff
@@ -36,6 +43,9 @@ def read_storage(self, contract_address, index):
 
     def dynld(self, contract_address, dependency_address):
 
+        if not self.contract_loading:
+            raise ValueError("Cannot load contract when contract_loading flag is false")
```
Comment (JoranHonig) [inline mythril/support/loader.py]: Why a value error here, and an Exception a while back?
In this case a specific exception might make sense, wdyt?

Comment (norhh) [reply mythril/support/loader.py]: This exception is being caught in lot of other areas in laser, so we might need to import this exception from mythril to laser, should we do that?

**Comment by b-mueller** [comment]
Sorry for messing up the formatting, you were quicker than me fixing it again :)

**Pull request merged**